### DISCUSSION TITLE
Add support for Twitch Chat Pronouns

### DISCUSSION THREAD

**Discussion by ianmdesign** [Chatterino/chatterino2#3192] [Ideas] [answered]
https://pronouns.alejo.io/

**Comment by Felanbird** [comment]
We've implemented a Nickname feature that can be used to set users pronouns, we've discussed adding the extension before and we decided it was too niche. 
Also from a fork I've tested the API calls seem to be very delayed.

**Reply by apa420** [reply]
Can you link the fork so they can try it out? I can't seem to find it.

**Reply by Felanbird** [reply]
https://github.com/GabeEddyT/chatterino2/releases/tag/v2.3.3-pro1

**Reply by ianmdesign** [reply]
I'll echo some previous comments on this. The pronoun extension is becoming more widely used on twitch. It will improve interactiveness within many communities, particularly streamers who want to use Chatterino to read chat and moderate, but want to see the pronouns of their community members. 

API call being delayed makes sense, though it is not necessarily accurate to say the pronouns extension is niche.

**Reply by Felanbird** [reply]
> It will improve interactiveness within many communities,

This part has already been well handled by nicknames, communities I'm in which use pronouns have had no issue at all, and have in fact enjoyed, asking people their pronouns so they can add them to nicknames.


> particularly streamers who want to use Chatterino to read chat and moderate, but want to see the pronouns of their community members.

This is a more fair point, while community members can easily set-up pronouns for others, it would be more of a problem for streamers in this case. That being said if the streamers are small enough usage of the nickname feature would still be warranted. 

> though it is not necessarily accurate to say the pronouns extension is niche.

We deemed it to be too niche almost 1 year ago, obviously as time passes it's likely for things to pickup steam/gain users, similar to why we didn't immediately jump into supporting 7tv.

**Reply by ALazyMeme** [reply]
By adding the pronoun extension, you now have to add yet another setting into the already clustered settings menu. I think the nicknames feature is the best feature for the request.

**Comment by apa420** [comment, marked as answer]
#2319

**Reply by killertofus** [reply]
Duplicate of #2227
